addons/website_sale/views/templates.xml

```diff
@@ -1744,7 +1746,7 @@
                             <div t-if="product_image._name == 'product.image' and product_image.embed_code" class="d-flex align-items-center justify-content-center h-100 embed-responsive embed-responsive-16by9">
                                 <t t-raw="product_image.embed_code"/>
                             </div>
-                            <div  t-else="" t-field="product_image.image_1920" class="d-flex align-items-center justify-content-center h-100" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920", "itemprop": "image"}'/>
+                            <div  t-else="" t-field="product_image.image_1920" class="d-flex align-items-center justify-content-center h-100" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920"}'/>
```

Comment:
Is this change really necessary?

Reply:
I'm pretty sure it is. If it is not removed, then there will be two DOM elements with the attribute "itemprop"="image" (this one + the one I added) and from testing I noticed that this one will be picked by the schema validator. Even if it wasn't picked, its value is still wrong, since it doesn't contain the base_url.

Reply:
Then it was already broken since all images would have had the itemprop attribute if there are multiple images to display. 😬